infrastructure/helm/quantumserverless/README.md

```diff
@@ -15,6 +15,8 @@ helm dependency build
 Update values.yaml file. Find and replace the following strings
 
 - **CLIENTSECRET-CHANGEME**: string used as the secret for a OIDC protocol
+- **APISERVERSECRET-CHANGEME**: string used as the secret for a OIDC protocol for apiserver
+- **SECRET-CHANGEME**: string used as the secret for a OIDC protocol
 - **HELM-RELEASE**: release name used in the helm install command
 - **LOCAL-IP**: IP address that can be accessed from both outside of the cluster and inside of the cluster.  
```

Comment:
can you add Kind to the local-ip examples just below this?

Reply:
OK.  I will.  It's a little bit tricky one :-).

Reply:
I think a note saying to get the IP from the kind control plane is probably enough (i.e. `kubectl get nodes -o wide` and then use the internal IP for that pod, which should be `172.18.0.2`)